.env.test

```
@@ -0,0 +1,29 @@
+GIT_BASE_BRANCH=main
+HOST_APP_PORT=3000
+HOST_PRIVATE_APP_PORT=9080
+
+# app
+APP_VERSION=1
+GIT_COMMIT_SHA=sha
+APP_PORT=3000
+APP_BIND_ADDRESS=0.0.0.0
+APP_ENV=production
+LOG_LEVEL=info
+API_ADDRESS_DITTO=https://api.dittowords.com
```

dotenv-linter (3.3.0)

[warning] [UnorderedKey] The API_ADDRESS_DITTO key should go before the APP_BIND_ADDRESS key

(UnorderedKey)